src/box-client.ts

```diff
@@ -288,10 +288,13 @@ class BoxClient {
 		this.devicePins = new DevicePins(this);
 		this.webhooks = new Webhooks(this);
 		this.recentItems = new RecentItems(this);
-		this.collaborationWhitelist = new CollaborationWhitelist(this);
+		this.collaborationAllowlist = new CollaborationAllowlist(this);
 		this.termsOfService = new TermsOfService(this);
 		this.storagePolicies = new StoragePolicies(this);
 
+		// Legacy insensitive language
```

Comment:
Will this work for methods like `this.collaborationWhitelist.getAllWhitelistedDomains`? It seems like it will be `this.collaborationWhitelist.getAllAllowlistedDomains`, which means there still may be a breaking change.

Reply:
@sujaygarlanka I think it will because `CollaborationWhitelist` now includes these additional aliases:

```js
// Legacy insensitive language
(CollaborationAllowlist.prototype as any).getWhitelistedDomain =
	CollaborationAllowlist.prototype.getAllowlistedDomain;
(CollaborationAllowlist.prototype as any).getAllWhitelistedDomains =
	CollaborationAllowlist.prototype.getAllAllowlistedDomains;
```